html-practice/facebook-mock-up.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<style>
+		html {
```

Comment:
Little nitpick -- most people would indent one more here since it's inside the `<style>` tags.